SPECS/PyYAML/PyYAML.spec

```diff
@@ -59,6 +55,12 @@ chmod a-x examples/yaml-highlight/yaml_hl.py
 %{python3_sitelib}/*
 
 %changelog
+* Tue Nov 07 2023 Pawel Winogrodzki <pawelwi@microsoft.com> - 5.4.1-1
+- Upgrade to 5.4 to fix CVE-2020-1747 and CVE-2020-14343.
+
+* Fri Oct 27 2023 Xiaohong Deng <xiaohongdeng@microsoft.com> - 5.2-1
```

Comment:
This is the state from the in-progress monthly release. Once it's merged and `fasttrack/2.0` is updated, the diff should only contain one new entry.